src/followers/followers.service.ts

```diff
@@ -48,7 +52,8 @@ export class FollowersService {
                 followingId
             }
         })
-        return followUser;
+        const action = await this.actionsService.addAction('Followed',followedById,'User',followUser.id,followUser)
```

Comment:
here you have hadcoded strings. Store them in variables and reuse? insted of hardcode them